packages/core/src/extensions/Blocks/nodes/Block.ts

```diff
@@ -130,10 +130,10 @@ export const Block = Node.create<IBlock>({
 
           return true;
         },
-      // Deletes a block at a given position.
+      // Deletes a block at a given position and sets the selection to where the block was.
       BNDeleteBlock:
         (posInBlock) =>
-        ({ state }) => {
+        ({ state, view }) => {
```

Comment:
I think if we want to do things the tiptap / PM way, we should check for "dispatch" and don't issue the transactions if false?

Reply:
Yep, agreed, looking deeper into the PM docs it does seem to add functionality that we want.